entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -274,7 +312,8 @@ class EntityHandlerTests {
             .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
             .exchange()
             .expectStatus().isOk
-            .expectBody().json("{\"testedAt\":{\"type\":\"Property\",\"value\":{\"type\":\"Time\",\"@value\":\"11:20:30\"}},\"@context\":\"$NGSILD_CORE_CONTEXT\"}")
+            .expectBody()
+            .json("{\"testedAt\":{\"type\":\"Property\",\"value\":{\"type\":\"Time\",\"@value\":\"11:20:30\"}},\"@context\":\"$NGSILD_CORE_CONTEXT\"}")

```

Comment:
This one bothers me because we lose the fact that `.json(...)` applies to the body (so is directly related to the previous line). Strangely enough, it is correctly formatted in the next diff.

Reply:
You are right, I agree that in this kind of cases you need to force the formatting. The fact that in the next diff it was correctly formatted is because there were new line before for the first arg, there not.